### ISSUE TITLE
Tweak breakpoint mixins

### ISSUE THREAD

**Issue by johanneskrtek** [zotero/publications#40]
Minor issue: I didn't have a closer look by now, but I noticed that the `from()` mixin in `_respond.scss` adds 1px, which leads to a min-width of 769px for tablets – so the iPad would not be included.

I guess we could just not add that 1px?

**Comment by tnajdek** [comment]
Certainly, happy to revisit. Do you reckon having overlapping `from` and `upto` makes sense?

**Comment by johanneskrtek** [comment]
No, we should avoid overlapping.

E.g.: 'tablet': 768

`upto` => `tablet` – 1px
`from` => `tablet`
`between` => `small`, `tablet` – 1px

(Similar to the Bootstrap 3 logic)
